CogniteSdk.Types/Assets/AssetCreate.cs

```diff
@@ -53,6 +54,12 @@ public class AssetCreate
         /// </summary>
         public string ParentExternalId { get; set; }
 
+        /// <summary>
+        /// List of labels to associate with the asset.
+        /// </summary>
+        [Obsolete("The Label attribute is in development, and currently only available for use in playground")]
+        public List<Label>? Labels { get; set; }
```

Comment:
Should this be nullable? Then it's still a nullable reference type which we don't want (for now).